libp2p/observedaddrmanager.nim

```diff
@@ -33,14 +32,18 @@ proc getProtocol(self: ObservedAddrManager, observations: seq[MultiAddress], mul
   countTable.sort()
   var orderedPairs = toSeq(countTable.pairs)
   for (ma, count) in orderedPairs:
-    let maFirst = ma[0].get()
-    if maFirst.protoCode.get() == multiCodec and count >= self.minCount:
+    let
+      maFirst = ma[0].valueOr: continue
```

Comment:
how about sth like `ma[0].optValue.map((maFirst) => maFirst.protoCode).valueOr: continue`

Reply:
This was a bit hard to get to compile, LMKWYT

Reply:
great!